azbus/sender.go

```diff
@@ -150,3 +150,56 @@ func (s *Sender) Send(ctx context.Context, message *OutMessage) error {
 	log.Debugf("Sending message id %s took %s", id, time.Since(now))
 	return nil
 }
+
+func (s *Sender) NewMessageBatch(ctx context.Context) (*OutMessageBatch, error) {
+	return s.sender.NewMessageBatch(ctx, nil)
+}
+
+// BatchAddMessage calls Addmessage on batch
+// Note: this method is a direct pass through and exists only to provide a
+// mockable interface for adding messages to a batch.
+func (s *Sender) BatchAddMessage(batch *OutMessageBatch, m *OutMessage, options *azservicebus.AddMessageOptions) error {
```

Comment:
slight smell here imho, since this really doesn't belong to Sender as in comment above

Reply:
agree, let me see what I can do